lisa/schema.py

```diff
@@ -471,6 +482,21 @@ class DiskOptionSettings(FeatureSettings):
             allow_none=True, decoder=search_space.decode_count_space
         ),
     )
+    disk_controller_type: Optional[
+        Union[search_space.SetSpace[DiskControllerType], DiskControllerType]
+    ] = field(  # type:ignore
+        default_factory=partial(
+            search_space.SetSpace,
+            items=[
+                DiskControllerType.SCSI,
```

Comment:
Put both in items, so it's by default enabled for all.

Reply:
I was a bit confused about the use-case for this default. I thought it was the default for the runbook, in which case by default you want SCSI. For default of VM capability, I thought that was the purpose of the generate_maximum_capability function?

Any guidance?

Reply:
Fixed

Reply:
If the default value has only SCSI, it may not be able to deploy with NVMe by default. If put both of them here, as the priority list, SCSI will be default.